src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -25,7 +25,9 @@
                   = template.description
         - if project.packages.empty?
           %p
-            There are currently no base image templates available for project <strong>#{title_or_name(project)}</strong>.
+            There are currently no base image templates available for project
+            = succeed '.' do
```

Comment:
Append `.` at the end of the title or name project :)

Doc: [succeed](http://www.rubydoc.info/github/haml/haml/Haml%2FHelpers%3Asucceed)